include/dali/c_api.h

```diff
@@ -341,6 +347,13 @@ DLL_PUBLIC void
 daliCopyTensorNTo(daliPipelineHandle *pipe_handle, void *dst, int n, device_type_t dst_type,
                   cudaStream_t stream, int non_blocking);
 
+
+/**
+ * @brief Gets direct access to the dali buffer containing an output of the pipeline
+ */
+DLL_PUBLIC void daliOutputPtr(daliPipelineHandle* pipe_handle, int n,
```

Comment:
I agree. I was confused with `n` as well. I can change to `output_idx` if we decide to move forward with this PR

Reply:
`output_idx` seems to be perfect

Reply:
On the second though, I'm not actually convinced that we need this function. Maybe it would be better to leave that one for now and add it later if necessary, together with proper testing?